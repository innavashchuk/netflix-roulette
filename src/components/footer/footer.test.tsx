import * as React from 'react';
import {render, screen} from '@testing-library/react';
import Footer from './footer';

describe('Footer', () => {
    it('should initialize', () => {
        const {asFragment} = render(<Footer/>);
        expect(asFragment()).toMatchSnapshot();
    });

    it('should contain logo', () => {
        render(<Footer/>);
        expect(screen.findByText('netflixroulette')).toBeTruthy();
    });
});
