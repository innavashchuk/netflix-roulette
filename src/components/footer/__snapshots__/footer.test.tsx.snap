// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should initialize 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p
      class="logo"
    >
      <span
        class="logo__span"
      >
        netflix
      </span>
      roulette
    </p>
  </footer>
</DocumentFragment>
`;
